packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -18,11 +18,14 @@ async function getPathEnv(appRoot: string) {
   const RNIDE_PATH_DELIMITER = "{RNIDE_PATH_DELIMITER}";
 
   const shellPath = process.env.SHELL ?? "/bin/zsh";
-  const { stdout } = await execa(shellPath, [
-    "-i",
-    "-c",
-    `cd "${appRoot}" && echo "${RNIDE_PATH_DELIMITER}$PATH${RNIDE_PATH_DELIMITER}"`,
-  ]);
+  const { stdout } = await execa(
+    shellPath,
+    ["-i", "-c", `cd "${appRoot}" && echo "${RNIDE_PATH_DELIMITER}$PATH${RNIDE_PATH_DELIMITER}"`],
+    {
+      extendEnv: false,
```

Comment:
can you add comment here explaining this option as it is rarely used and not seen in other places in our codebase, therefor may be good to be able to understand it in case someone needs to copy this code for some other purpose